# goal definition
geometry_msgs/PointStamped lastKnownPosition
bool anyLastPosition
---
---

